packages/grafana-ui/src/components/LoadingBar/LoadingBar.tsx

```diff
@@ -0,0 +1,43 @@
+import { css, keyframes } from '@emotion/css';
+import React from 'react';
+
+/**
+ * @internal
+ */
+export interface LoadingBarProps {
+  containerWidth: number;
+  width?: number;
+  height?: number;
+  ariaLabel?: string;
+}
+
+/**
+ * @internal
+ */
```

Comment:
Hmm, I am not convinced that we should mark this as "internal".

My main reasoning is that everything we introduce in the public `@grafana/ui` package is basically public and there is a good chance that people will start depending on it, even if it is marked as "internal". This means that we have to maintain it in the future and that we cannot change it easily. 

In case we would only use it in the core for now I would suggest to keeping it in there and only "expose it" once we are convinced that it's in a mostly stable state.

Reply:
Yes, that's right - the maintenance of this shared component might prove to be more of a problem than needed. We will discuss taking it out of grafana/ui for now 🤲.

Reply:
🙏

Reply:
Hi @leventebalogh and @grafana/dashboards-squad - we moved `LoadingBar` outside of the `grafana/ui`, but this creates an unnecessary problem: [`PanelChrome` in `grafana/ui`](https://github.com/grafana/grafana/blob/ui/loading-bar/packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx#L56) needs the `LoadingBar` and will be used in the dashboard page and outside of grafana core. The dashboard page can pass the `LoadingBar` to `PanelChrome` because it is in core, but then no one else can use the same bar unless they copy it. 
Another problem is that the loading state of the panel and the loading component rendered for this loading state become two separate props to `PanelChrome` that make no sense without each other. So they have to always be present together, or not at all.

As is it right now the `LoadingBar` is self-sufficient and @axelavargas and I discussed it would be better to maintain it inside `grafana/ui` if needed instead of complicating the API of the `PanelChrome` which needs it. That said, the **internal** comment is definitely unnecessary and is removed.